contracts/cw1-whitelist/src/contract.rs

```diff
@@ -74,9 +74,11 @@ where
     if !can_execute(deps.as_ref(), &info.sender)? {
         Err(ContractError::Unauthorized {})
     } else {
-        let mut res = Response::default();
-        res.messages = msgs;
-        res.attributes = vec![attr("action", "execute")];
+        let res = Response {
+            messages: msgs,
+            attributes: vec![attr("action", "execute")],
+            ..Default::default()
```

Comment:
I know they are the same, but I prefer `Response::default` to `Default::default` when we have access to the explicit types (not just traits)